bpf/process/generic_calls.h

```diff
@@ -1206,6 +1213,88 @@ FUNC_INLINE int generic_retkprobe(void *ctx, struct bpf_map_def *calls, unsigned
 	return 1;
 }
 
+FUNC_INLINE int generic_retuprobe(void *ctx, struct bpf_map_def *calls, unsigned long ret)
```

Comment:
I think this could be shared by both retkprobe/retuprobe as generic_retprobe with few ifdefs noted below

the socket related code would be ifdef GENERIC_KRETPROBE